include/pybind11/numpy.h

```diff
@@ -1368,22 +1401,33 @@ template <typename T, int Flags> struct handle_type_name<array_t<T, Flags>> {
 
 NAMESPACE_END(detail)
 
-template <typename Func, typename Return, typename... Args>
-detail::vectorize_helper<Func, Return, Args...>
-vectorize(const Func &f, Return (*) (Args ...)) {
-    return detail::vectorize_helper<Func, Return, Args...>(f);
-}
```

Comment:
Moved this into `detail` as it didn't seem to be an intentionally exposed part of the API.